java/feature-layer-dictionary-renderer/README.md

```diff
@@ -1,19 +1,36 @@
 # Feature Layer Dictionary Renderer
 
-Displays a FeatureLayer with military symbology, using ArcGIS Runtime's DictionaryRenderer.
+Convert features into graphics to show them with mil2525d symbols.
 
-![Feature Layer Dictionary Renderer App](feature-layer-dictionary-renderer.png)
+![Image of feature layer dictionary renderer](feature-layer-dictionary-renderer.png)
+
+## Use case
+
+A dictionary renderer uses a style file along with a rule engine to display advanced symbology. This is useful for displaying features using precise military symbology.
+
+## How to use the sample
+
+Pan and zoom around the map. Observe the displayed military symbology on the map.
 
 ## How it works
 
-This sample loads a number of point, line, and polygon feature tables from a Runtime geodatabase. For each feature table, a FeatureLayer is created, and a DictionaryRenderer object is created and applied to the layer. Note that each layer needs its own renderer, though all renderers can share the DictionarySymbolStyle, in which case all layers will use the same symbology specification (MIL-STD-2525D in the case of this sample). Each layer is added to the map, and when all layers are loaded, the map's viewpoint is set to zoom to the full extent of all feature layers.
+1. Create a `Geodatabase` using `Geodatabase(geodatabasePath)`.
+1. Load the geodatabase asynchronously using `Geodatabase.loadAsync()`.
+1. Instantiate a `SymbolDictionary`  using `DictionarySymbolStyle(dictionaryPath)`.
+1. Load the symbol dictionary asynchronously using `DictionarySymbol.loadAsync()`.
+1. Wait for geodatabase to completely load by connecting to `Geodatabase.addDoneLoadingListener()`.
+1. Cycle through each `GeodatabaseFeatureTable` from the geodatabase using `Geodatabase.getGeodatabaseFeatureTables()`.
+1. Create a `FeatureLayer` from each table within the geodatabase using `FeatureLayer(GeodatabaseFeatureTable)`.
+1. Load the feature layer asynchronously with `FeatureLayer.loadAsync()`.
+1. Add the feature layer to map using `Map.getOperationalLayers().add(FeatureLayer)`.
+1. Create `DictionaryRenderer(SymbolDictionary)` and attach to the feature layer using `FeatureLayer.setRenderer(DictionaryRenderer)`.
+1. Wait for each layer to load using `FeatureLayer.addDoneLoadingListener`.
```

Comment:
probably don't need to specify to wait for each layer to load using addDoneLoadingListnener.